fea-rs/src/compile/variations.rs

```diff
@@ -17,32 +15,29 @@ use write_fonts::{
 /// This trait abstracts over that info.
 pub trait VariationInfo {
     /// If the tag is an axis in this font, return the min/max values from fvar
-    fn axis_info(&self, axis_tag: Tag) -> Option<AxisInfo>;
-
-    /// Return the normalized value for a user coordinate for the given axis.
-    ///
-    /// NOTE: This is only used for computing the normalized values for ConditionSets.
-    ///
-    /// NOTE: if unit suffixes become a thing, and ConditionSets remain a thing, then
-    /// this should use the same AxisLocation enum as below.
-    fn normalize_coordinate(&self, axis_tag: Tag, value: Fixed) -> F2Dot14;
+    fn axis(&self, axis_tag: Tag) -> Option<(usize, &Axis)>;
```

Comment:
I'd clarify the docs to say that it returns the axis as well as the index in of the axis as it will appear in fvar (and elsewhere?)

Reply:
Good catch, adding.